server/test/buff_server/accounts_test.exs

```diff
@@ -84,4 +83,6 @@ defmodule BuffServer.AccountsTest do
       assert %Ecto.Changeset{} = Accounts.change_user(user)
     end
   end
+
+  setup :verify_on_exit!
```

Comment:
i understand the logic. but that's not really the convention.
even though the name is :verify_on_exit, all setup calls are run before the tests, also, i usually not read all the tests, and scrolling to the bottom of the file to find an intializer makes reasoning about the flow harder IMO

Reply:
👍